src/lib.rs

```diff
@@ -1,6 +1,27 @@
+use elements::{BlockHash, Txid};
+
+pub mod avg_vbytes;
+pub mod client;
+mod coin_selection;
+mod estimate_transaction_size;
 pub mod input;
 pub mod loan;
 pub mod oracle;
 pub mod swap;
+pub mod wallet;
 
-mod estimate_transaction_size;
+// todo!(Make our own UTXO type, this is a http payload)
+#[derive(serde::Deserialize, Debug, PartialEq, Clone, Copy)]
+pub struct Utxo {
+    pub txid: Txid,
+    pub vout: u32,
+    pub status: UtxoStatus,
+}
+
+#[derive(serde::Deserialize, Debug, PartialEq, Clone, Copy)]
+pub struct UtxoStatus {
+    pub confirmed: bool,
+    pub block_height: Option<u64>,
+    pub block_hash: Option<BlockHash>,
+    pub block_time: Option<u64>,
+}
```

Comment:
:question: Should we move this to the `wallet` module? Maybe `client` and `coin_selection` could be moved there too.